packages/cli/src/lingui-compile.ts

```diff
@@ -52,7 +49,7 @@ function command(config, options) {
 
     catalogs.forEach((catalog) => {
       const messages = catalog.getTranslations(
-        locale === config.pseudoLocale ? config.sourceLocale : locale,
```

Comment:
this change means that we will pseudolocalize messages in the order as you were describing

1. translation
2. fallback (using the new fallbackLocales)
3. message key

before we were automatically using sourceLocale

Reply:
Nice, thank you!